subscription/test/access_control_test.py

```diff
@@ -8,84 +8,104 @@
 
 class TestAccessControl(unittest.TestCase):
     def setUp(self):
-        self.ac = AccessControl()
+        self.access_control = AccessControl()
 
-    @patch.dict(os.environ, {"ENVIRONMENT_NAME": "test_env"})
+    @patch.dict(os.environ, {"ACCESS_CONTROL_URL": "http://localhost:3011/access-control"})
+    def test_get_url_from_parameter_store_local(self):
+        self.access_control.get_url_from_parameter_store()
+        self.assertEqual(self.access_control.url, "http://localhost:3011/access-control")
+
+    @patch.dict(os.environ, {"ENVIRONMENT_NAME": "SIT"})
     @patch('access_control.Env_Vars')
-    def test_get_url_from_parameter_store(self, mock_env_vars):
-        mock_env_vars.get_var.side_effect = [
-            "https", "443", "example.com", "api/v1"
+    def test_get_url_from_parameter_store_aws(self, mock_env_vars):
+        mock_env_vars_instance = MagicMock()
+        mock_env_vars_instance.get_var.side_effect = [
+            "https", "3011", "cmr.sit.earthdata.nasa.gov", "/access-control"
         ]
-
-        self.ac.get_url_from_parameter_store()
-
-        self.assertEqual(self.ac.url, "https://example.com:443/api/v1")
-
-    @patch.dict(os.environ, {})
-    def test_get_url_from_parameter_store_no_env(self):
-        with self.assertRaises(ValueError):
-            self.ac.get_url_from_parameter_store()
-
-    @patch('access_control.AccessControl.get_url_from_parameter_store')
-    def test_get_url(self, mock_get_url):
-        mock_get_url.return_value = None
-        self.ac.url = "https://test.com"
-
-        # Exercise
-        result = self.ac.get_url()
-
-        # Verify
-        self.assertEqual(result, "https://test.com")
-        mock_get_url.assert_not_called()
-
-    @patch('access_control.AccessControl.get_url_from_parameter_store')
-    def test_get_url_not_set(self, mock_get_url):
-        mock_get_url.return_value = None
-        self.ac.url = None
-
-        # Exercise
-        self.ac.get_url()
-
-        # Verify
-        mock_get_url.assert_called_once()
-
-    @patch('access_control.requests.get')
-    @patch('access_control.AccessControl.get_url')
-    def test_get_permissions_success(self, mock_get_url, mock_requests_get):
-        mock_get_url.return_value = "https://test.com"
+        mock_env_vars.return_value = mock_env_vars_instance
+
+        self.access_control.get_url_from_parameter_store()
+        expected_url = "https://cmr.sit.earthdata.nasa.gov:3011/access-control"
+        self.assertEqual(self.access_control.url, expected_url)
+
+    def test_get_url(self):
+        with patch.object(AccessControl, 'get_url_from_parameter_store') as mock_method:
+            self.access_control.url = "http://test-url.com"
```

Comment:
that url is not a dead URL but it does not return either. Switch to example.gov or example.com

Reply:
I have updated the URL